src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -219,4 +221,35 @@ public static TaskStatus parseStatus(String status) throws ParseException {
         }
         return TaskStatus.of(trimmedStatus);
     }
+
+    /**
+     * Parses a {@code String description} into a {@code ExamDescription}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code description} is invalid.
+     */
+    public static ExamDescription parseExamDescription(String description) throws ParseException {
+        requireNonNull(description);
+        String trimmedDescription = description.trim();
+        if (!ExamDescription.isValidDescription(trimmedDescription)) {
+            throw new ParseException(ExamDescription.DESCRIPTION_CONSTRAINTS);
+        }
+        return new ExamDescription(trimmedDescription);
+    }
+
+    /**
+     * Parses a {@code String examDate} into a {@code ExamDate}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code examDate} is invalid.
+     */
+    public static ExamDate parseExamDate(String examDate) throws ParseException {
```

Comment:
Might want to standardise this method with Douglas' `parseDeadlineTag` method, since both have to deal with dates. Currently, the ExamDate constructor takes in a string argument, while the DeadlineTag constructor takes in LocalDate argument. Perhaps can consider standardising this for code consistency?